### DISCUSSION TITLE
Find and report applicable backward-incompatible changes when Core update becomes availabile

### DISCUSSION THREAD

**Discussion by ejz32983** [frenck/spook#845] [Ideas]
Would it be possible to have the change log reviewed to identify any backward-incompatible changes that would affect any existing integrations, devices, entities, or automations, then report them as a notification? It would be nice if it could also include suggestions on how to resolve them. Would there be any risks that would make this a bad idea?

**Comment by frenck** [comment]
> Would it be possible to have the change log

Backwards incompatible changes should not be logged, but raise repair warning. Unless the backwards incompatible change is from a developer nature (in that case, it is a message for the custom integration developer).

This is not really useful for the user and can also not be identified as such a change.

../Frenck
